### ISSUE TITLE
Add Wi-Fi setup to settings

### ISSUE THREAD

**Issue by rock3r** [squanchy-dev/squanchy-flutter#49]
The settings screen should display a "configure conference Wi-Fi" item if the device runs iOS 11+ and the app has Wifi configuration data. If the hotspot is already configured on the device, it should be marked as "already done".

⚠️ **There is still no designs for this story**

⚠️ Blocked by #48 (implementing the WiFi setup code)

🗒 Wi-Fi setup is only available on [iOS 11+](https://developer.apple.com/documentation/networkextension/nehotspotconfiguration)

🗒 This setting can only be shown if the app has the WiFi configuration data, otherwise it must be hidden.
